Student/StudentUITests/Assignments/IPadAssignmentsTests.swift

```diff
@@ -112,6 +111,9 @@ class IPadAssignmentsTest: MiniCanvasUITestCase {
         pointsTextAssignment.add(submission: makeTextSubmission(score: 13))
 
         pullToRefresh()
+        sleep(5)
+        pullToRefresh()
```

Comment:
This change was necessary, without waits the ui was flaky, looked like sometimes the minicanvas wasn't updated by the time the refresh happened